static/src/javascripts/bootstraps/commercial.js

```diff
@@ -55,7 +56,8 @@ if (!commercialFeatures.adFree) {
         ['cm-stickyTopBanner', initStickyTopBanner],
         ['cm-paidContainers', paidContainers],
         ['cm-paidforBand', initPaidForBand],
-        ['cm-commentAdverts', initCommentAdverts]
+        ['cm-commentAdverts', initCommentAdverts],
+        ['rr-adblock-ask', initAdblockAsk]
```

Comment:
Is this code also meant to run across `dotcom-rendering` articles too? If so you'll need to add it to `/bootstraps/dotcom-rendering-commercial.js`.

Reply:
Thanks for the note about this George! I think we'll make a note to add this at a later date.